projects/m1/025-caesar-cipher/python/main.py

```diff
@@ -0,0 +1,14 @@
+message = input('insert a message to encode or decode: ')
+number = int(input('insert a positive number to encode or negative to decode: '))
+
+ALPHABET = 'abcdefghijklmnopqrstuvwxyz'
+new_message = ''
+
+
+for i in message:
+    for e in ALPHABET:
+        if e == i:
+            index = ALPHABET.index(e) 
+            new_message += ALPHABET[index + number]
+                
+print(new_message)
```

Comment:
I recommend trying to use (input) numbers greater than 60.
If there are errors, how could you fix the code?